### ISSUE TITLE
[DrDump] MPC-HC crashed because of fault in Intel graphics driver

### ISSUE THREAD

**Issue by clsid2** [clsid2/mpc-hc#2805]
This page opened because MPC-HC crashed due to a fault in the Intel graphics driver. This specific crash may happen when using EVR renderer and filter graph is closed.

THIS IS NOT A BUG IN THE PLAYER!

The following solutions might help:

**Solution 1:**

Disable unwanted video processing in the Intel graphics control panel.

The Intel driver settings have some options for video post-processing. These options are often enabled by default. They are known to cause playback and stability problems and are also useless for improving video quality.

It is recommended to disable these options: _Contrast Enhancement_ and _Skin Tone Enhancement_.

![intel_driver_settings](https://user-images.githubusercontent.com/4704996/131339797-959ad3a2-1f1d-44bd-bef5-c418997d1490.png)

**Solution 2:**

Install a newer version of the Intel graphics driver.
https://downloadcenter.intel.com/product/80939/Graphics

**Solution 3:**

Change video renderer:
MPC-HC options > Playback > Output
Use MPC Video Renderer which you can get at our [download page](https://github.com/clsid2/mpc-hc/releases).

Additionally make this change:
MPC-HC options > Internal Filters > Video decoder > set "Hardware decoder" to "D3D11"

Also disable seekbar preview (which uses EVR as renderer).
MPC-HC options > Player > User Interface > Hover type = Seek time

**Solution 4:**

Restart the graphics driver by pressing:
Win + Ctrl + Shift + B
(Windows 10/11 only)

**Comment by clsid2** [comment]
Feedback is appreciated if one of these solutions works, or if you manage to find another solution. This can help others with same problem.

**Comment by VicTheVicar** [comment]
This error started to appear after a Windows update. I get it when I try to open another video. Some videos are more likely to crash the player than others. First I upgraded the player to the newest version. I also changed from CCCP to K-Lite. No change. 

1. I've turned off all the "enhancements" in the Intel Control Panel. No change.

2. I've already got the latest driver, and Intel doesn't seem to update them?

3a. I changed to K-lite from CCCP. Downloaded the player + K-lite package and installed over. The option "MPC Video Renderer" doesn't exist. I got "Enhanced Video Rendering (custom presenter)" - current setting, "Enhanced Video Rendering", "Sync Rendering", "Video Mixing Rendering 9 (renderless)", "Video Mixing Rendering 9 (windowed)", "Video Mixing Rendering 7", "Null (anything)" and "Null (uncompressed)" 
  b. I don't have "Hardware decoder" but "DXVA2 (Copy-back)", "DXVA2 (native)" - original setting, "D3D11" - current setting, "NVIDIA CUVID (old)" and "Intel QuickSync (old)".
  c. Was already set to Seek Time.
